setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = apig-wsgi
-version = 2.7.0
+version = 2.8.0
```

Comment:
please don't bump versions in this commit, that is done separately at release time.

Reply:
Apologies. Will revert.